pkg/web/static/js/main.js

```diff
@@ -24,3 +24,21 @@ if (logoutBttn) {
     localStorage.removeItem(hideWelcomeMsg)
   })
 }
+
+document.body.addEventListener('htmx:responseError', (e) => {
+  // Close any open modals.
```

Comment:
When modals are open the toast appears behind it. This makes it easy for users to not notice them. I attempted to resolve this by setting the z-index really high, but that didn't work. We can return to this with follow on stories, as I wanted to ensure this gets added.